src/types.ts

```diff
@@ -0,0 +1,18 @@
+import {Observable} from './Observable';
```

Comment:
I think this file (`types.ts`) is not landed in current TOT of codebase, isn't it?